com.woltlab.wcf/templates/email_html.tpl

```diff
@@ -155,11 +155,13 @@
 	{capture assign='footer'}
 	{hascontent}
 	<span style="font-size: 0;">-- <br></span>
-	{content}
-	{@MAIL_SIGNATURE|language|newlineToBreak}
-	{/content}
-	{/hascontent}
-	{/capture}
+{content}
+{if MAIL_SIGNATURE_HTML|language}
+{@MAIL_SIGNATURE_HTML|language}
+{else}
+{@MAIL_SIGNATURE|language|newlineToBreak}
+{/if}
+{/content}{/hascontent}{/capture}
```

Comment:
```suggestion
	{content}
	{if MAIL_SIGNATURE_HTML|language}
	{@MAIL_SIGNATURE_HTML|language}
	{else}
	{@MAIL_SIGNATURE|language|newlineToBreak}
	{/if}
	{/content}{/hascontent}{/capture}
```

Reply:
This *was* intentional to get rid of the initial tabs in case significant whitespace was used (e.g. `<pre>`). But that doesn't actually make sense, because the entirety of the signature is generated by the constant, I probably was confused by the `{if}` logic.

Thanks.